### PR TITLE
fix(console): Edit section issues

### PR DESCRIPTION
**Pull request by yasinmiran**
### Related Issues
(None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation
- [ ] Documentation provided. (Add links)
- [ ] Unit tests provided. (Add links if any)
- [ ] Integration tests provided. (Add links if any)

### Related PRs
- (None)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 1]
```diff
@@ -2716,15 +2716,14 @@ export const console: ConsoleNS = {
                 },
                 dangerZoneGroup: {
                     deleteIDP: {
-                        actionTitle: "Delete Identity Provider",
+                        actionTitle: "Delete",
```
Comment (brionmario) [inline modules/i18n/src/translations/en-US/portals/console.ts:2719]: We would need to do this change for `Applications` as well.

cc:/ @ayshsandu

File: apps/console/src/extensions/configs/identity-provider.ts [hunk 2]
```diff
@@ -44,6 +45,28 @@ export const identityProviderConfig: IdentityProviderConfig = {
...
                 IdentityProviderManagementConstants.SESSION_EXECUTOR_AUTHENTICATOR
             ].includes(name);
+        },
+        isProvisioningAttributesEnabled(authenticatorId: string): boolean {
+            const excludedAuthenticators = new Set([]);
+            /**
+             * If the {@param authenticatorId} is not in the excluded set we
+             * can say the provisioning attributes is enabled for authenticator.
+             */
+            return !excludedAuthenticators.has(authenticatorId);
```
Comment (brionmario) [inline apps/console/src/extensions/configs/identity-provider.ts:55]: Let's return `false` here and add this as a doc comment.

Comment (yasinmiran) [reply apps/console/src/extensions/configs/identity-provider.ts:55]: This has been addressed in commit c8571bd.

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/identity-providers/components/forms/components/common-pluggable-component-form.tsx [hunk 3]
```diff
@@ -207,8 +207,106 @@ export const CommonPluggableComponentForm: FunctionComponent<CommonPluggableComp
...
+         *      - Why form state represents API's structure?
+         *
+         * This is what happens when you abstract things to the point where you can't
+         * implement a basic logical condition that was invented in 1918. Below this point
+         * you will see a specific logic tied to SAML protocol fields. But this is the
+         * only way we can ensure that values have the proper state in the form based on
+         * the current state.
+         *
+         * PS: np thank me later! append(:joy)
+         */
```
Comment (DimalChandrasiri) [inline apps/console/src/features/identity-providers/components/forms/components/common-pluggable-component-form.tsx:266]: Please remove this comment.

File: apps/console/src/features/identity-providers/components/forms/components/common-pluggable-component-form.tsx [hunk 4]
```diff
@@ -207,8 +207,106 @@ export const CommonPluggableComponentForm: FunctionComponent<CommonPluggableComp
...
+
+        if (key === "IsLogoutReqSigned" || key === "ISAuthnReqSigned") {
```
Comment (DimalChandrasiri) [inline apps/console/src/features/identity-providers/components/forms/components/common-pluggable-component-form.tsx:268]: Move the string to constant for better code quality.

**Pull request merged**